Update device-control-removable-storage-access-control.md (#5493)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md

   When Type **AuditAllowed** or **AuditDenied** is selected:

   - 0: nothing
   - 1: show notification, only works for AuditDenied
   - 2: send event

>    - 3: show notification and send event. If apply this to AuditAllowed, will only fire event for reporting but not have notification.

Reviewer: ```suggestion
   - 3: show notification and send event. If applying this to AuditAllowed, will only fire the event for reporting but will not show the notification.
```

   
**Property name: Sid**

> Description: Defines whether apply this policy over specific user or user group; one entry can have maximum one Sid and an entry without any Sid means applying the policy over the machine.

Reviewer: ```suggestion
Description: Defines whether to apply this policy over a specific user or user group; one entry can have a maximum of one SID and an entry without any SID means applying the policy over the machine.
```

**Property name: ComputerSid**

> Description: Defines whether apply this policy over specific machine or machine group; one entry can have maximum one ComputerSid and an entry without any ComputerSid means applying the policy over the machine. If you want to apply an Entry to a specific user and specific machine, add both Sid and ComputerSid into the same Entry.

Reviewer: ```suggestion
Description: Defines whether to apply this policy over a specific machine or machine group; one entry can have a maximum of one ComputerSID and an entry without any ComputerSID means applying the policy over the machine. If you want to apply an entry to a specific user and specific machine, add both SID and ComputerSID into the same entry.
```
